terraform/modules/self-service/db.tf

```
@@ -1,7 +1,7 @@
 resource "aws_db_instance" "self_service" {
   name           = "selfservice"
   engine         = "postgres"
-  engine_version = "10.6"
+  engine_version = "10.10"
```

```suggestion
  engine_version = "11.5"
```
https://deployer.tools.signin.service.gov.uk/teams/main/pipelines/verify-self-service/jobs/deploy-to-production/builds/175#L5e29c042:146

---

Hmmm. I wonder why production is running a major version newer version of Postgres compared to staging. I guess we can just do this change but it seem odd...

---

they were provisioned at different times (and the default version was different?).. would be my guess 🤷‍♂